src/runtime_src/core/common/query_requests.h

```diff
@@ -2534,6 +2523,21 @@ struct is_ready : request
   get(const device*) const = 0;
 };
 
+struct is_ready_msg : request
```

Comment:
Please add a comment as to what this query request does.

Reply:
I added a comment for the ready struct above. I ended up deleting the `is_ready_msg` struct as it was not needed.